### ISSUE TITLE
Auto-hyphens show as tofu in Chromium-based browsers

### ISSUE THREAD

**Issue by hugovk** [bitprophet/alabaster#187]
* macOS Monterey
* Chrome 95
* Chrome Canary 98
* Opera 71 (also Chromium)

The CSS uses `hyphens: auto` which inserts hyphens for some words, however with the hyphen shows as an empty box aka ["tofu"](https://en.wikipedia.org/wiki/Specials_(Unicode_block)#Replacement_character):

![image](https://user-images.githubusercontent.com/1324225/141686571-ac0b13f7-2c78-448d-acd3-bb82c1e747b5.png)

https://alabaster.readthedocs.io/en/latest/changelog.html

Non-Chromium browsers such as Firefox 94 (pictured here) and Safari 15.1 are fine:

![image](https://user-images.githubusercontent.com/1324225/141685929-8d0b24d8-ac14-442b-86a7-a5346dc972a5.png)

Also reported to Chromium: https://bugs.chromium.org/p/chromium/issues/detail?id=1270110

**Comment by hugovk** [comment]
Update: this was a Chromium bug on macOS Monterey, confirmed fixed in Chrome Canary 98.0.4742.0:

* https://bugs.chromium.org/p/chromium/issues/detail?id=1267606

Due for Chrome stable release on 1st Feb 2022:

* https://www.chromestatus.com/features/schedule

Closing.

**Comment by bitprophet** [comment]
Just ran into this myself (finally left Mojave in the dust...) - thanks for reporting it and posting the followup!

**Comment by hugovk** [comment]
Update: Chrome 98 has now been released and confirmed this is fixed (Version 98.0.4758.80). 👍

**Comment by bitprophet** [comment]
Confirmed! Awesome. This was getting frustrating. Thanks for the followup. Now if only this Chrome version fixed its Github Enterprise caching issues at my dayjob... 😭

**Comment by luckman212** [comment]
I'm the original author of the chromium bug https://bugs.chromium.org/p/chromium/issues/detail?id=1267606

I just want to note that it is _not_ fixed, even as of version 101.0.4951.41 Official

**Comment by hugovk** [comment]
Hi Luke, thanks for the followup!

Yes, your bug is still open, but [the one I opened](https://bugs.chromium.org/p/chromium/issues/detail?id=1270110) was merged into yours (maybe it shouldn't have been), and that has been fixed.

I can no longer reproduce the specific problem above with Alabaster in Chrome 100.0.4896.127 on macOS: https://alabaster.readthedocs.io/en/latest/changelog.html

So for Alabaster, I think we're good here, unless it can be reproduced? Hope your bug gets fixed soon too!
